src/lib/pagination/ctrl.coffee

```diff
@@ -0,0 +1,99 @@
+EventsEmitter = require( 'events' )
+Pagination = require( './index' )
+PaginationView = require( './view' )
+
+###
+ * #PaginationCtrl
+ * Creates a pagination element and controls it.
+ *
+ * ## Events
+ *
+ * | CONSTANT.argument  | Description |
+ * |--------------------| -------------|
+ * | CHANGE | When page number is changed by user |
+ * | CHANGE.current  | New page number |
+ * | CHANGE.last | Number of last page |
+###
+class PaginationCtrl extends EventsEmitter
```

Comment:
Yes but I just dont want to allow access to @pagination and I want the class to have its own event api